pom.xml

```diff
@@ -13,6 +13,44 @@
     <artifactId>inamik-text-tables</artifactId>
     <version>0.8</version>
 
+    <properties>
+        <!--General -->
+        <maven.compiler.target.version>1.8</maven.compiler.target.version>
+        <maven.compiler.source.version>1.8</maven.compiler.source.version>
+        <required.java.version>1.8</required.java.version>
```

Comment:
What is your reasoning for dropping Java 1.6 support?

Although not documented, supporting Java 1.6 is intentional.

Reply:
Actually we don't need 1.8 and I have not changed much except except some diamond operators here and there. Will revert to 1.6 accordingly.